npm install from github 会直接报错

执行 `npm install git://github.com/aliyun/fun.git#master` 会报错
ERROR in ./index.js
Module not found: Error: Can't resolve 'uuid' in 'xxxxx'

原因是 fun 下的 package.json 的 prepack 脚本是 `make package-fun-nas-server`
在这个过程中会去 webpack fun-nas-server, 这个过程出错了.

--- a/lib/utils/fun-nas-server/package.json
+++ b/lib/utils/fun-nas-server/package.json
@@ -19,14 +19,15 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "ali-oss": "^6.3.1",
     "@webserverless/fc-express": "^0.1.4",
+    "ali-oss": "^6.3.1",
     "body-parser": "^1.19.0",
     "express": "^4.17.0",
     "express-async-errors": "^3.1.1",
     "fs-extra": "^8.1.0",
     "md5-file": "^4.0.0",
     "raw-body": "^2.4.1",
-    "stream": "0.0.2"
+    "stream": "0.0.2",
+    "uuid": "^7.0.2"
   }
 }

